components/chat_widget/src/components/ocs-chat/ocs-chat.tsx

```diff
@@ -309,6 +323,31 @@ export class OcsChat {
     this.parsedStarterQuestions = this.parseJSONProp(this.starterQuestions, 'starter questions');
   }
 
+  private async initializeTranslations() {
+    let customTranslationsObj: Partial<TranslationStrings> | undefined;
+
+    if (this.translationsUrl) {
+        customTranslationsObj = await this.loadTranslationsFromUrl(this.translationsUrl);
+    }
+
+    this.currentLanguage = this.language || getBrowserLanguage();
```

Comment:
`TranslationManager` already falls back to the browser language if the provided language is empty so this line isn't necessary